src/josepy/jws.py

```diff
@@ -252,21 +262,23 @@ class JWS(json_util.JSONObjectWithFields):
 
```

Comment:
Can we update this incorrect docstring in this or another PR? You didn't change it, but you figured out what the types actually are here.

(On a larger level, I wonder if there's something we that can be done to reduce the duplication between our type annotations and the types in our docstrings.)

Reply:
Yes here and in the `acme`  module, I just gave up editing every docstring that was either with wrong types arguments, missing arguments or invalid return, because it would increase the PR size without an immediate value (well, everyone is living with this documentation right ?).

So we can tackle the documentation issue in a further PR. During my study some potential toolings between the type annotations and the docstring, so I hope we will be able to fix most of the inconsistencies in an automated way.